stdpopsim/slim_engine.py

```diff
@@ -214,34 +286,31 @@
 
 
 def slim_makescript(
-        script_file,
-        trees_file,
-        demographic_model,
-        samples,
-        Q,
-        check_coalescence,
-        mutation_rate,
-        generation_time,
-        population_configurations,
-        recombination_map,
-        migration_matrix,
-        demographic_events,
-        verbosity):
+        script_file, trees_file,
+        demographic_model, contig, samples,
+        Q, check_coalescence, verbosity):
```

Comment:
Q isn't the most self explanatory parameter name, any chance of something more descriptive?